wealth-watch-script.sql

```diff
@@ -0,0 +1,35 @@
+create table user
+(
+id int unsigned not null unique auto_increment,
+name varchar(45) not null unique,
+password varchar(20) not null,
+primary key (id)
+);
+
+create table portfolio
+(
+id int unsigned not null unique auto_increment,
+user_id int unsigned not null,
+name varchar(50) not null unique,
+currency enum('CAD', 'USD', 'BRL') not null,
+primary key (id),
+constraint fk_portfolio_user foreign key (user_id) references user (id)
```

Comment:
Lets make the constraint-creating pattern like `CONSTRAINT fk_currenttable_foreigntable_id`... which in this case would be `CONSTRAINT fk_portfolio_user_id`...

Reply:
done on both constraints (tables portfolio and portfolio_item).